### PR TITLE
Some more improvements to documentation

### PR DESCRIPTION
**Pull request by rootkea**
The `gallery` shortcode needs to be in `<div>` otherwise the accompanying attribute text doesn't start on next line.

Also why do we need `get_url(path=asset) | safe` in `gallery.html` I don't understand...

### CODE REVIEW

File: docs/content/documentation/content/image-processing/index.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ The source for all examples is this 300 pixel × 380 pixel image:
 
 ## Using `resize_image` in markdown via shortcodes
 
-`resize_image` is a built-in Tera global function (see the [templates](@/documentation/templates/_index.md) chapter),
```
Comment (Keats) [inline docs/content/documentation/content/image-processing/index.md:124]: It's useful to remind them this is a Tera function, I would not remove that

File: docs/content/themes/float/index.md [hunk 2]
```diff
@@ -46,7 +46,8 @@ Float 是一款為 [Zola](https://www.getzola.org/) 設計的佈景主題。
 在您的 Zola 專案資料夾內：
 
 把 Float 以 Git 子模組的方式加入專案內：
-```shell
+
+```sh
```
Comment (Keats) [inline docs/content/themes/float/index.md]: This is from a theme so it will get overriden the next time a theme is updated and I should probably not include non-English themes...

**Review by Keats — CHANGES_REQUESTED** [review]

File: docs/content/documentation/content/image-processing/index.md (hunk 1 above)
Comment (rootkea) [reply docs/content/documentation/content/image-processing/index.md:124]: This is a bit tricky. The current doc says that `resize_image()` is a built-in Tera function but https://tera.netlify.app/docs/#built-in-functions has no `resize_image()` since it's a built-in Zola function. Ref - https://www.getzola.org/documentation/templates/overview/#built-in-global-functions

Comment (Keats) [reply docs/content/documentation/content/image-processing/index.md:124]: Yeah it's a bit hard to parse. It's a built-in (by Zola) Tera function

Comment (rootkea) [reply docs/content/documentation/content/image-processing/index.md:124]: Okay. So how about this:
```
`resize_image` is a Zola built-in global function written in Tera template engine (see the [templates](@/documentation/templates/_index.md) chapter),
```

Comment (Keats) [reply docs/content/documentation/content/image-processing/index.md:124]: `resize_image` is a Zola built-in Tera function (see the [templates](@/documentation/templates/_index.md) chapter)

?

Comment (rootkea) [reply docs/content/documentation/content/image-processing/index.md:124]: Sure!

**Comment by rootkea** [comment]
I noticed few other issues with the docs. I'll add them and shall make this PR ready. :)

**Comment by Keats** [comment]
@rootkea anything more you want to do or is this ready?

**Comment by rootkea** [comment]
Sorry, I didn't have much time at hand. I'll look into this tomorrow. :)

**Comment by rootkea** [comment]
I'll open separate PR if I found any further documentation issues. I think we should merge the work done till now. Please let me know if something needs to be changed. :)

**Review by Keats — APPROVED** [review]

**Comment by Keats** [comment]
Can you resolve the conflicts?

**Comment by rootkea** [comment]
@Keats Done! Sorry, I took much time with this PR.

But I intend to resume contributing to Zola more actively. :)

**Review by Keats — APPROVED** [review]
